logger/unilog.go

```diff
@@ -189,6 +190,7 @@ type tagPair struct {
 // independentTags stores a list of tags to individually emit metrics on.
 // Each metric name will be formatted exactly once and cached in metricsTable.
 type independentTags struct {
+	sync.RWMutex
```

Comment:
This isn't a huge deal, but I'd generally recommend having this be an unexported variable on the struct, rather than an embedded type. Embedded types are pretty specific in their uses.

In this case, either works, but my rule of thumb is: use unexported variables unless there's a clear reason to use embedded types instead.